plugin/trino-faker/src/main/java/io/trino/plugin/faker/FakerConnector.java

```diff
@@ -126,6 +126,17 @@ public List<PropertyMetadata<?>> getSchemaProperties()
                         null,
                         defaultLimit -> checkProperty(1 <= defaultLimit, INVALID_SCHEMA_PROPERTY, "default_limit value must be equal or greater than 1"),
                         false),
+                doubleProperty(
+                        SchemaInfo.SEQUENCE_MIN_DISTINCT_VALUES_RATIO,
+                        """
+                                Minimum ratio of distinct values of a column to total number of rows in a table to treat the columns as a sequence
+                                when creating a table in this schema using existing data. Set to a value greater than 1 to disable using sequences""",
+                        null,
+                        sequenceMinDistinctValuesRatio -> checkProperty(
+                                0 <= sequenceMinDistinctValuesRatio && sequenceMinDistinctValuesRatio <= 2,
```

Comment:
Why would distinct values to row count ratio be higher than 1 ?

Reply:
The description explain it - to disable detecting sequences.

Reply:
We shouldn't allow a whole range of impossible values (1, 2] just to have a way of disabling.

Reply:
I'll change this into a boolean value.